### PR TITLE
bug/FPHS-370

### PR DESCRIPTION
**Pull request by dephillipsmichael**
Fixed core motion permission bug by making it retain its value thought initialization

### CODE REVIEW

File: APCAppCore/APCAppCore/Library/Permissions/APCPermissionsManager.m [hunk 1]
```diff
@@ -80,8 +85,14 @@ - (instancetype)init
         
         [[NSNotificationCenter defaultCenter] addObserver:self selector:@selector(appDidRegisterForRemoteNotifications:) name:APCAppDidRegisterUserNotification object:nil];
 
-        _coreMotionPermissionStatus = kPermissionStatusNotDetermined;
-                
+        // Make sure coreMotionPermissionStatus is in a valid state, and it isn't overwritten
```
Comment (Erin-Mounts) [inline APCAppCore/APCAppCore/Library/Permissions/APCPermissionsManager.m]: I'd suggest instead just initializing the static variable with kPermissionStatusNotDetermined, and then not resetting it here at all...

Comment (dephillipsmichael) [inline APCAppCore/APCAppCore/Library/Permissions/APCPermissionsManager.m]: good point, fixed that

**Comment by EricSiegNW** [comment]
looks good to me with latest update

**Comment by Erin-Mounts** [comment]
agree, merging

**Pull request merged**